webhooks/structures.go

```diff
@@ -252,7 +252,7 @@ type BotCreated struct {
 	MaxChatsCount        *uint                        `json:"max_chats_count,omitempty"`
 	DefaultGroupPriority configuration.GroupPriority  `json:"default_group_priority,omitempty"`
 	Groups               []*configuration.GroupConfig `json:"groups,omitempty"`
-	WorkScheduler        configuration.WorkScheduler  `json:"work_scheduler,omitempty"`
+	WorkScheduler        *configuration.WorkScheduler `json:"work_scheduler,omitempty"`
```

Comment:
```suggestion
	WorkScheduler        configuration.WorkScheduler `json:"work_scheduler,omitempty"`
```
In v3.4 `configuration.WorkScheduler` is a map, which is a pointer field already so there's no need to use pointer here

Reply:
fixed in [API-12351: apply review suggestions](https://github.com/livechat/lc-sdk-go/pull/144/commits/3bdbd1fbf9fad7510a48c7d8f26b5d6531691e32)